docs_user/modules/proc_pulling-configuration-from-a-tripleo-deployment.adoc

```diff
@@ -20,16 +26,16 @@ connection=ssh
 remote_config_path=/tmp/tripleo
 ----
 +
-Make sure the ssh command you provide in `ssh_cmd` parameter is correct and with key authentication.
+Ensure that the ssh command you provide in `ssh_cmd` parameter is correct and includes key authentication.
```

Comment:
How do users ensure that the ssh command you provide in the `ssh_cmd` parameter is correct and includes key authentication?

Reply:
The simplest way is by running the command to make sure it's working